.github/workflows/ray-docker.yml

```diff
@@ -0,0 +1,51 @@
+name: ray-docker
+on: pull_request
+jobs:
+  test-ray-docker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - shell: bash -l {0}
+        run: bash examples/docker/ci-ray-docker/build-docker-image.sh
+      - shell: bash -l {0}
+        run: docker create --name ray_container ray_docker_image:1.1.0 bash -c "tail -f /dev/null"
+      - shell: bash -l {0}
+        run: docker start ray_container
+      - shell: bash -l {0}
+        run: docker exec ray_container pytest modin/experimental/xgboost/test/test_default.py
+      - shell: bash -l {0}
+        run: docker exec ray_container pytest modin/pandas/test/dataframe/test_binary.py
+      - shell: bash -l {0}
+        run: docker exec ray_container pytest modin/pandas/test/dataframe/test_default.py
+      - shell: bash -l {0}
+        run: docker exec ray_container pytest modin/pandas/test/dataframe/test_indexing.py
+      - shell: bash -l {0}
+        run: docker exec ray_container pytest modin/pandas/test/dataframe/test_iter.py
+      - shell: bash -l {0}
+        run: docker exec ray_container pytest modin/pandas/test/dataframe/test_join_sort.py
+      - shell: bash -l {0}
+        run: docker exec ray_container pytest modin/pandas/test/dataframe/test_map_metadata.py
+      - shell: bash -l {0}
+        run: docker exec ray_container pytest modin/pandas/test/dataframe/test_reduction.py
+      - shell: bash -l {0}
+        run: docker exec ray_container pytest modin/pandas/test/dataframe/test_udf.py
+      - shell: bash -l {0}
+        run: docker exec ray_container pytest modin/pandas/test/dataframe/test_window.py
+      - shell: bash -l {0}
+        run: docker exec ray_container python -m pytest modin/pandas/test/test_series.py
```

Comment:
is there a reason those run via `python -m pytest` while others above use just `pytest`?

Reply:
There is no actual reason for this, so just `pytest` calls were used.